src/commands/license/show.ts

```diff
@@ -4,9 +4,17 @@ import { outputFormat } from './util';
 export = {
 	command: 'show',
 	desc: 'Show active license',
+	builder: yargs => {
+		yargs.options({
+			'usage': {
+				desc: 'Include license usage',
+				type: 'boolean'
+			},
+		});
+	},
 	handler: argv => {
 		runCommand(argv, async (client, output) => {
-			const license = await client.licensing.get();
+			const license = await client.licensing.get(argv.usage && output.outputJson);
```

Comment:
Can we show a warning if `--usage` is specified without `--json`?
I tested this out, and was confused why `--usage` wasn't appending the querystring